### PR TITLE
feat: add action filter

### PR DESCRIPTION
**Pull request by dotennin**
# Summary

1. Action filter
image

2. line-height minor adjustments
image

### CODE REVIEW

File: src/utils.tsx [hunk 1]
```diff
@@ -76,3 +76,14 @@ export function getText(
     return type;
   }
 }
+
+export const debounce = <T extends (...args: unknown[]) => unknown>(
+  callback: T,
+  delay = 250,
+): ((...args: Parameters<T>) => void) => {
+  let timeoutId: NodeJS.Timeout
```
Comment (Aeolun) [inline src/utils.tsx:84]: I don't think this is a NodeJS timeout. It's probably just number.

Can use window.setTimeout instead to get the correct type I think.

File: src/StoreActions.tsx [hunk 2]
```diff
@@ -313,13 +352,15 @@ export const StoreActions = (props: {
...
                 .filter((item) => {
-                  console.log(
-                    item.action.type,
-                    item.diff && rootItem(item.diff, filterPath)
-                  );
+                  const {
+                    diff,
+                    action: { type: actionType },
+                  } = item;
+                  console.log(actionType, diff && rootItem(diff, filterPath));
```
Comment (Aeolun) [inline src/StoreActions.tsx:359]: We can probably remove this one ;)

File: src/StoreActions.tsx [hunk 3]
```diff
@@ -240,6 +263,13 @@ export const StoreActions = (props: {
     return getItemString(type, data, undefined, true, false);
   }, []);
 
+  const debounceSetAcitonFilter = useCallback(
```
Comment (Aeolun) [inline src/StoreActions.tsx]:
```suggestion
  const debounceSetActionFilter = useCallback(
```

**Review by Aeolun — APPROVED** [review]
LGTM, few comments, but nothing breaking.

File: src/utils.tsx (hunk 1 above)
Comment (dotennin) [reply src/utils.tsx:84]: Ohh yeah , should use window.setTimeout one

**Pull request merged**